src/background/hypothesis-chrome-extension.js

```diff
@@ -219,11 +219,15 @@ export default function HypothesisChromeExtension({
     state.clearTab(tabId);
   }
 
-  // installs or uninstalls the sidebar from a tab when the H
-  // state for a tab changes
+  /**
+   * installs or uninstalls the sidebar from a tab when the H
+   * state for a tab changes
+   *
+   * @param {chrome.tabs.Tab} tab
```

Comment:
The JSDoc params are inconsistent with the code here. `tab` is marked as not-null in the JSDoc type but then you're checking for the case where it is nullish below. Either we should avoid calling this function if we don't have a tab, or we should make it clear that the argument might be nullish and the function needs to handle that.

Reply:
Yes, I agree about the inconsistency.